docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -1,39 +1,46 @@
 include::variables.adoc[]
 
-[[set-up-driver-plugin]]
-= Setting Up the Driver Plug-in
+[[setup-virtualization-environment]]
+= Setting Up the Virtualization Environment
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 1
+:next-steps: Proceed to xref:../getting-started/installing.adoc#[Installing {project}] once your hypervisor has been installed and configured.
```

Comment:
I think we can get Next step as part of TOC as the last link after `Setting Up Minishift to Use VirtualBox` instead of adding in the middle of sections. That looks odd IMO.

WDYT @LalatenduMohanty @gbraad @praveenkumar 
See how it will look:
![screenshot from 2018-03-26 14-51-50](https://user-images.githubusercontent.com/1132451/37898005-45f96202-3105-11e8-8219-38a99d6d32cf.png)
![screenshot from 2018-03-26 14-52-02](https://user-images.githubusercontent.com/1132451/37898006-46335214-3105-11e8-9fb8-2481934b9986.png)

Reply:
I agree with @budhrg , in current form it looks odd.

Reply:
@LalatenduMohanty @budhrg I agree that it looks odd in **_getting-started/installing.adoc_**, but I believe that it helps with navigation in this document.

In this case, each user is most likely to follow the instructions for their specific operating system and have little need to consult any other. For this reason, we include the Next Steps following each installation procedure to allow for easier navigation of the process. This way, the user does not need to find where the next steps are or scroll past the instructions for operating systems that do not apply to them -- the next steps are simply presented directly following the setup they just performed.

Personally, I think the benefit in terms of navigation is important here. The reader is more likely to follow the instructions for their platform linearly and discover the next step intuitively when the information directly follows the setup for their platform. Simply having the next steps as part of the TOC by adding it to the end of the document adds a burden on the user to both recognize its placement and follow it accordingly if it's not presented linearly.